packages/compartment-mapper/test/scaffold.js

```diff
@@ -73,7 +73,7 @@ export function scaffold(
     });
   };
 
-  wrap(test)(`${name} / loadLocation`, async t => {
+  wrap(test.only)(`${name} / loadLocation`, async t => {
```

Comment:
debug line?

Reply:
now you know why I did wrap(test) instead of creating my own test function that calls test inside ;)